connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1889,6 +2087,10 @@ ble_error_t Gap::setAdvertisingData(
     bool scan_response
 )
 {
+    tr_info("Set advertising data for set %d", handle);
+#if TRACE_WRITE_VALUES
```

Comment:
Why the payload is logged separatly ? The parameters minimise fragmentation and scan response are important too.

Reply:
So that the user can disable it by setting `TRACE_WRITE_VALUES` to 0. I don't know how necessary this is though. I'll add logs for the minimise fragmentation and scan response parameters.